lib/MTT/Util.pm

```diff
@@ -490,4 +492,13 @@ sub term_handler{
 	Verbose("All is done, exiting.\n");
 	exit(0);
 }
+
+#--------------------------------------------------------------------------
+
+# Trim the leading and trailing whitespaces
+sub trim_spaces {
+    my ($str) = @_;
+    $str =~ s/^\s*|\s*$//g;
```

Comment:
In my mind they will do the same thing here. Would ``\s+`` be better form/style than ``\s*`` in this case?